content/en/platform/corda/5.0-beta/introduction/introduction.md

```diff
@@ -0,0 +1,42 @@
+---
```

Comment:
I think this is an error @Niamh25? We don't want Beta content in the GA branch.

Reply:
Yep this is an error, I removing it now!